web/src/components/organisms/Detail.jsx

```diff
@@ -0,0 +1,16 @@
+import { FaCalendarAlt, FaMapMarkerAlt } from 'react-icons/fa';
+
+const Detail = ({ label, value }) => {
```

Comment:
Can you rename this to 
```suggestion
const RowItem = ({ label, value }) => {
```